mzmine-community/src/main/java/io/github/mzmine/parameters/parametertypes/selectors/FeatureListsSelection.java

```diff
@@ -202,17 +202,17 @@ public FeatureListsSelection clone() {
     if (keepSelection) {
       newSelection.specificFeatureLists = specificFeatureLists;
       newSelection.batchLastFeatureLists = batchLastFeatureLists;
-      newSelection.evaluatedSelection = evaluatedSelection;
     }
+    newSelection.evaluatedSelection = evaluatedSelection;
```

Comment:
Why move this out of the keepSelection if branch?
There needs to be a way to copy parameters and reset the selection. 
This is the same behavior like in RawDataFilesSelection